packages/studio-ui/src/web/components/ContentForm/UploadWidget/index.tsx

```diff
@@ -39,9 +39,9 @@ const UploadWidget: FC<IUploadWidgetProps> = props => {
     setError(error)
   }
 
-  const onChange = (value: string | null) => {
+  const onChange = (value: string | null, allowEmpty: boolean = false) => {
     const error = validate(value)
```

Comment:
PP: It could be simplified like so. I mean you don't need to call validate when allowEmpty is set to true. But since validate is really simple you can keep it like that.

```suggestion
    if (!allowEmpty) {
      const error = validate(value)
    }
    // ...
```